### DISCUSSION TITLE
Unknown property 'enabled' for Report, gradle 8.1.0

### DISCUSSION THREAD

**Discussion by mohamed2020m** [gsantner/markor#2080] [General] [answered]
I encountered this error after upgrading to gradle 8.1.0 and trying to build the project:

```Could not set unknown property 'enabled' for Report junitXml of type org.gradle.api.internal.tasks.testing.DefaultJUnitXmlReport.```

the problem was in the build.grade(Project)

I resovle the issues by replacing the following two lines:

```
reports.junitXml.enabled = true
reports.html.enabled = true
```

with: 

```
reports{
    junitXml {
        enabled true
    }
}
reports{
    html {
        enabled true
    }
}
```

**Comment by mohamed2020m** [comment]
yes, you're right, but I was reading the code, and I got the gradle update notification from android studio, then I upgraded it to the newest version, but I after that when I was trying to rebuild the project, I got the error.

**Reply by gsantner** [reply, marked as answer]
Ah OK. Can you  please check if the changed solution still generates unit test reports? (= `make test`)

If so, please feel free to make a merge request to increase the version to the newer one and the required fixes for it.
It also can be seen at merge request CI build if it generates unit testing results still

**Reply by mohamed2020m** [reply]
Ok, I'll check it, thanks.

**Reply by gsantner** [reply]
Hello @mohamed2020m , did you check again?
